CHANGES.md

```diff
@@ -21,7 +21,11 @@
   - Add new package for `irmin` command-line tool (#1951, @metanivek)
 
 - **irmin-pack**
-  - Add a GC (TODO)
+  - Add a garbage collection feature, allowing a user to discard commits older
+    than a specified commit. See `examples/gc.ml` for a demonstration of usage.
+    This feature requires a new on-disk format; migration happens automatically
+    when opening a store as read-write after upgrading to this version of Irmin.
+    Additionally, to use GC a store must use `Indexing_strategy.minimal`.
```

Comment:
The 3.3 changelog already details the new IOs, so I'd maybe skip this part.

Reply:
Ah, thanks for that catch. I removed that part.